rmf_fleet_adapter/src/full_control/main.cpp

```diff
@@ -467,11 +467,26 @@ struct Connections : public std::enable_shared_from_this<Connections>
           path_request_pub, mode_request_pub);
 
     const auto& l = state.location;
+    const auto& starts = rmf_traffic::agv::compute_plan_starts(
+        *graph, state.location.level_name, {l.x, l.y, l.yaw},
+        rmf_traffic_ros2::convert(adapter->node()->now()));
+    if (starts.empty())
+    {
+      RCLCPP_ERROR(
+        adapter->node()->get_logger(),
+        "Unable to compute a StartSet for robot [%s]. This can happen if the "
```

Comment:
Let's also print out what we received for the robot's coordinates and map name. That may help users with debugging.

Reply:
Good point. Done https://github.com/open-rmf/rmf_ros2/pull/13/commits/0818cd5f2046d08675778391762f7931aa0e2db8